bg_atlasgen/validate_atlases.py

```diff
@@ -144,34 +152,16 @@ def validate_atlas(atlas_name, version, all_validation_functions):
     if not updated:
         update_atlas(atlas_name)
 
-    validation_function_parameters = [
-        # validate_atlas_files(atlas_path: Path)
-        (Path(get_brainglobe_dir() / f"{atlas_name}_v{version}"),),
-        # validate_mesh_matches_image_extents(atlas: BrainGlobeAtlas)
-        (BrainGlobeAtlas(atlas_name),),
-        # open_for_visual_check()
-        (),
-        # validate_checksum()
-        (),
-        # check_additional_references()
-        (),
-        # validate_mesh_structure_pairs(atlas_name: str, atlas_path: Path):
-        (
-            atlas_name,
-            Path(get_brainglobe_dir() / f"{atlas_name}_v{version}"),
-        ),
-    ]
-
     # list to store the errors of the failed validations
-    failed_validations = []
-    successful_validations = []
+    failed_validations = {atlas_name: []}
+    successful_validations = {atlas_name: []}
 
-    for i, validation_function in enumerate(all_validation_functions):
+    for i, validation_function in enumerate(validation_functions):
         try:
-            validation_function(*validation_function_parameters[i])
-            successful_validations.append((atlas_name, validation_function))
+            validation_function(atlas_name)
```

Comment:
I tried running the individual functions locally on "allen_mouse_100um". They all worked. When I tried validate_atlases it gave me an error. I think line 161 should be: validation_function(BrainGlobeAtlas(atlas_name)) instead of validation_function(atlas_name). It works when I change it.